src/renderer/components/Map.tsx

```diff
@@ -9,6 +10,7 @@ import { useTranslation } from 'react-i18next';
 import { EmptyClasses } from '../constants/animalsClasses';
 import AnimalsListTooltipContent from './AnimalsListTooltipContent';
 import styles from './Map.module.scss';
+import '@blueprintjs/popover2/lib/css/blueprint-popover2.css';
```

Comment:
Let's keep the imports organized in two groups separated by a blank line: external packages and app-specific imports. Inside each group let's keep the imports organized alphabetically (press Shift + Alt + O to "Organize Imports" in VSCode).

With this convention it is obvious where each import should go and the diffs are easier to review. :slightly_smiling_face:

Reply:
I just fixed it. Thank you for the shortcut 🙂